### PR TITLE
Add Ænix to the Adopters list

### PR DESCRIPTION
**Pull request by kvaps**
Hi, we utilize KubeVirt for running virtual machines and Kubernetes-as-a-Service in our free PaaS platform [Cozystack](https://cozystack.io/).

We'd like to share this information :)

### CODE REVIEW

**Comment by kvaps** [comment]
/assign @aburdenthehand 
/assign @maiqueb

👋

**Review by 0xFelix — APPROVED** [review]
/release-note-none
/lgtm

File: ADOPTERS.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
...
 | Integration | Kasten by Veeam | 2022 | [link](https://docs.kasten.io/latest/usage/openshift_virtualization.html?highlight=kubevirt) | [Kasten K10](https://www.kasten.io/product/) manages KubeVirt and Red Hat OpenShift Virtualization VMs seamlessly for policy based automation of backup, disaster recovery, and application mobility across different Kubernetes versions, distributions, storage providers, and clouds. |
 | Integration | Trilio | 2021 | [link](https://trilio.io/) | TrilioVault has been cloud-native since day one and protects the most demanding environments to maximize stability across all tenants. Our platform is built for flexibility across deployments, integrating seamlessly with Kubernetes, OpenStack and Red Hat Virtualization. That means we can equally backup and restore stateful and stateless applications based on VMs, Containers or VMs in Containers based on KubeVirt.  |
+| Vendor | Ænix | 2023 | [link](https://aenix.io/) | Ænix provides consulting services for cloud providers and uses KubeVirt in free PaaS platform [Cozystack](https://cozystack.io) for running virtual machines and Kubernetes-as-a-Service. |
```
Comment (fabiand) [inline ADOPTERS.md:23]: Can we drop the `provides consulting services` part?

Comment (kvaps) [reply ADOPTERS.md]: @fabiand sure, done

**Review by xpivarc — COMMENTED** [review]
/approve

**Comment by dhiller** [comment]
:stop_sign: please refrain from manual retesting.

:warning: this PR is being held to stop automatic retesting. Quarantine PR https://github.com/kubevirt/kubevirt/pull/11244 is given priority.

/hold

**Comment by dhiller** [comment]
:warning: removing hold since https://github.com/kubevirt/kubevirt/pull/11244 got merged.

/unhold

**Pull request merged**